dockerfiles/hack/run-integration.py

```diff
@@ -31,15 +31,15 @@
 # Messages to stdout
 STREAM_HANDLER = logging.StreamHandler(sys.stdout)
 STREAM_HANDLER.setFormatter(logging.Formatter(fmt='%(message)s'))
-LOG.addHandler(STREAM_HANDLER)
 
 # Messages to the log file
 if LOG_FILE is not None:
     FILE_HANDLER = logging.FileHandler(LOG_FILE)
     FILE_HANDLER.setFormatter(logging.Formatter(fmt='%(message)s'))
-    LOG.addHandler(FILE_HANDLER)
 
-LOG.setLevel(logging.INFO)
+# Setting up the root logger
+logging.basicConfig(level=logging.INFO,
+                    handlers=(STREAM_HANDLER, FILE_HANDLER))
```

Comment:
I have a question. What if LOG_FILE is None? could handlers contain an undefined one?

Reply:
I don't think so. Good catch! Feel free to fix it :)